interfaces.go

```diff
@@ -55,21 +65,19 @@ type MapTable interface {
 // Multimap recipe
 //
 
-// MultimapTable lets you list rows based on a field equality, eg. 'list all sales where seller id = v'.
 type MultimapTable interface {
-	Set(v interface{}) Op
-	Update(v, id interface{}, m map[string]interface{}) Op
-	Delete(v, id interface{}) Op
-	DeleteAll(v interface{}) Op
-	List(v, startId interface{}, limit int, pointerToASlice interface{}) Op
-	Read(v, id, pointer interface{}) Op
-	MultiRead(v interface{}, ids []interface{}, pointerToASlice interface{}) Op
+	Set(value interface{}) Op
+	Update(value, id interface{}, updatedValues map[string]interface{}) Op
+	Delete(value, id interface{}) Op
+	DeleteAll(value interface{}) Op
+	List(partitionKey, clusteringKey interface{}, limit int, pointerToASlice interface{}) Op
+	Read(partitionKey, clusteringKey, pointer interface{}) Op
+	MultiRead(partitionKey interface{}, ids []interface{}, pointerToASlice interface{}) Op
 	WithOptions(Options) MultimapTable
 	Table() Table
 	TableChanger
 }
 
-// MultimapMkTable lets you list rows based on several fields equality, eg. 'list all sales where seller id = v and name = 'john'.
 type MultimapMkTable interface {
 	Set(v interface{}) Op
 	Update(v, id map[string]interface{}, m map[string]interface{}) Op
```

Comment:
Should we improve this interface too, and others?

Reply:
Sorry, I should've clarified that I'm fixing `MapTable`, `MultimapTable`, `MultimapMultiKeyTable `TimeSeriesTable` first and will deal with the others a little later :)